README.md

```diff
@@ -50,6 +50,8 @@ Usage
 - Checkout the [guide](doc/PortingGuide.md) how to port your project.
 - Checkout the [tuning guide](doc/TuningGuide.md) for a step further to performance
   portable code.
+- Checkout the [interoperability guide.md](doc/InteroperabilityGuide.md) to understand
+  how to use **cupla** with software developed with an **alpaka** compatible interface.
```

Comment:
```suggestion
- Checkout the [interoperability guide](doc/InteroperabilityGuide.md) to learn more on
  how to use **cupla** with software developed with an **alpaka** compatible interface.
```
It looks irritating with the file extension, because there is no file with that name :)